src/components/Table/Table.jsx

```diff
@@ -183,6 +184,7 @@ export const defaultProps = baseProps => ({
     hasResize: false,
     useAutoTableLayoutForResize: false,
     shouldLazyRender: false,
+    wrapCellText: undefined,
```

Comment:
From the PR body you mentioned
> If wrapText is not explicitly defined we apply it only for tables with normal columns*. For tables with fixed or resizable columns we use truncation.

`undefined` as default will cause this PR to be a breaking change. This will modify the existing default behavior of the table. Applications with automated testing around the table may have tests fail based on this prop. When upgrading they will have to modify their code to state `wrapCellText: true` in order to preserve the existing behavior.

If I understand correctly -  to preserve the existing default functionality, we could just have the default be `true` instead?

Reply:
Yes, using true would preserve current default. We can do that but I implemented this as a change of default behaviour on purpose since I interpreted this comment that way 
https://github.ibm.com/Watson-IoT/pal-tracking/issues/331#issuecomment-18867586

It is possible that I misunderstood that comment, in that case we should definitively have `wrapCellText: true` as the default value like you say.

Reply:
with the latest PR change, default would be `wrapCellText: always`

Reply:
Yeah, a default of `wrapCellText: always` will preserve the existing default behavior. I believe the comment you linked is invalid - to my knowledge we've always had wrapping as the default.

Reply:
The designers have asked for the truncation to be the new default but I don't think they would be willing to wait for a major  version release to get it. @lukefirth @JordanWSmith15 are we ok with keeping with the opt in so that we can get this feature published?

Reply:
I've now changed the default to be `wrapCellText: always` and updated tests and stories.

Reply:
Hey - apologies i'm so late in the game to this stuff. Maximo App Suite is a killer....

There's generally a _lot_ that we'll want to sit back and consider about what a major version update of the tables should contain/consider with all the new capabilities we are building etc and possible changes to default behaviors etc, to the point that I'd like us to maybe start actively considering a table current, vs next major release version and consider that, potentially even putting a date to the next major release version.

In the meantime, I'd like us to make use of the design level to catch and create consistency as long as the implemented components can support the various capabilities. 

Meaning: I'm happy for us to retain the current default behaviors in the component today, as long as when designs require it to be different (eg: truncate over wrap) they can be honored without a lot of code work on the implementer. Once we reach a state where _most_ designs are using truncate by default (via standardisation at the design level), meaning most implementers are making that change, which hopefully from now we will see happen. Then at that point, it will only make logical sense to use the next available major update to change the components behavior to reflect the new default (along with other major changes). 

A timeline to that next major release will help us understand what other major changes should, and can be bundled into it.

Reply:
@lukefirth We have an open umbrella issue for v3.0.0 - #956 

Things like this can be added there so we have a running list.